lib/config/localSettings.js

```diff
@@ -8,8 +8,12 @@ module.exports = {
 	 * @param {ParsoidConfig} parsoidConfig
 	 */
 	setup: function(parsoidConfig) {
+		if (!process.env.ENV) {
+			return; // no environment given - parser tests?
```

Comment:
Why would we bypass generic setup for the unknown environments?

Reply:
We utilize some customizations in the config to support loading wiki configuration on demand based on domain passed in the URL. This doesn't play well with parser tests, which try to set up their own mock domain config.

Reply:
Got it.